MonoGame.Framework.Content.Pipeline/OggImporter.cs

```diff
@@ -24,5 +25,10 @@ public override AudioContent Import(string filename, ContentImporterContext cont
             var content = new AudioContent(filename, AudioFileType.Wav);
             return content;
         }
+
+        public override AudioContent Import(Stream input, string virtualFilename, ContentImporterContext context)
+        {
+            throw new NotSupportedException("OGG content must be stored on disk, not imported from a stream.");
```

Comment:
Being pedantic here, but the correct name is "Ogg", not "OGG".  It's not an acronym.